lib/process-argv.js

```diff
@@ -0,0 +1,20 @@
+function getProcessArgvBinIndex () {
+  // Built Electron app: app argv1 argv2 ... argvn
+  // (see https://github.com/electron/electron/issues/4690#issuecomment-217435222)
+  if (process.defaultApp === false) return 0
```

Comment:
is `process.defaultApp` set by electron, probably worth noting this in the comment as well.

Reply:
OK, I'll do it ASAP

Reply:
Comment added